tests/builtins/test_abs.py

```diff
@@ -8,7 +8,8 @@ class NotAbsLike:
                 pass
             x = NotAbsLike()
             print(abs(x))
-            """)
+            print('Done.')
+            """, exits_early=True)
```

Comment:
This should probably be a change to the test, catching and verifying that the right exception is raised.